packages/nodes-base/credentials/Kafka.credentials.ts

```diff
@@ -29,6 +29,68 @@ export class Kafka implements ICredentialType {
 			type: 'boolean',
 			default: true,
 		},
+		{
+			displayName: 'SSL CA',
+			name: 'sslCa',
+			type: 'string',
+			displayOptions: {
+				show: {
+					ssl: [true],
+				},
+			},
+			default: '',
+		},
+		{
+			displayName: 'Ignore SSL Issues',
```

Comment:
Let's follow conventions from other nodes with SSL auth

```suggestion
			displayName: 'Ignore SSL Issues (Insecure)',
			description: 'Whether to connect even if SSL certificate validation is not possible',
```